.github/integ-config/utils.yml

```diff
@@ -0,0 +1,6 @@
+minimal_browser_list:
+  - chrome
+  - firefox
+extended_browser_list:
+  - chrome
+  - firefox
```

Comment:
clever. I like the dry/isolation here. Might as well add safari to the extended list now. Its okay to go ahead and configure everything now, and fix the failing parts in future PR. Until we merge, fine to move fast and break things a bit.